src-electron/zcl/zcl-loader-dotdot.js

```diff
@@ -456,10 +456,11 @@ function prepareCluster(cluster, types, isExtension = false) {
  */
 function prepareAtomic(type) {
   let desc = type.$.name
+  let length = type.restriction?.[0]?.['type:length']?.[0]?.$?.value
```

Comment:
This is pretty complicated.

Reply:
Due to the format of the XML object this is how deep the length property is. Would you suggest creating a helper for this?